src/DynamoCore/Core/CustomNodeManager.cs

```diff
@@ -922,6 +923,7 @@ public bool TryGetNodeInfo(string name, out CustomNodeInfo info)
                         // two kinds of nodes whose input type are available:
                         // function node and custom node. 
                         List<Library.TypedParameter> parameters = null;
+
```

Comment:
Looks like a lot of dead codes are sitting in this method, please help with some cleanup and documentation of this method in your followup PR.